app/assets/stylesheets/layout/_layout.scss

```diff
@@ -45,6 +45,17 @@ body {
       margin-bottom: 40px;
     }
   }
+
+}
+
+.registrations .box, .sessions .box{
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space